packages/devtools_app/lib/src/shared/config_specific/url/_url_web.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be found
 // in the LICENSE file.
 
+// ignore: avoid_web_libraries_in_flutter, as designed
```

Comment:
would it be possible to ignore this lint for any file under the config_specific directory?

Reply:
I tried, but it did not work: https://github.com/dart-lang/linter/issues/4308